test/Ocelot.AcceptanceTests/PollyQoSTests.cs

```diff
@@ -180,11 +180,11 @@ private void GivenThereIsAServiceRunningOn(string url, int statusCode, string re
             });
         }
 
-        public void Dispose()
-        {
-            _serviceHandler?.Dispose();
+        public void Dispose()
+        {
+            _serviceHandler?.Dispose();
             _steps.Dispose();
-            GC.SuppressFinalize(this);
-        }
-    }
+            GC.SuppressFinalize(this);
+        }
+    }

```

Comment:
Exclude this file from PR! Zero changes and these tests are not related to the feature changes...

Reply:
Maybe merge is responsible?